packages/afp/.github/ISSUE_TEMPLATE/performance.md

```
@@ -0,0 +1,14 @@
+---
+name: Performance Update
+about: A code change that improves performance
+title: "perf: "
+labels: performance
+---
+
+**Description**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)